app/src/main/kotlin/com/wire/android/ui/home/settings/SettingsScreen.kt

```diff
@@ -103,29 +109,48 @@ private fun LazyListScope.folderWithElements(
         items = items.associateBy { it.id }
     ) { settingsItem ->
         SettingsItem(
-            item = settingsItem,
-            clickable = remember { Clickable(enabled = true) { onItemClicked(settingsItem) } }
+            title = settingsItem.title.asString(),
+            onRowPressed = remember { Clickable(enabled = true) { onItemClicked(settingsItem) } }
         )
     }
 }
 
 @Composable
-private fun SettingsItem(item: SettingsItem, clickable: Clickable) {
+fun SettingsItem(
+    title: String,
+    @DrawableRes trailingIcon: Int? = null,
+    onRowPressed: Clickable = Clickable(false),
+    onIconPressed: Clickable = Clickable(false),
+    modifier: Modifier = Modifier
+) {
     RowItemTemplate(
         title = {
-            Text(
-                style = MaterialTheme.wireTypography.body01,
-                color = MaterialTheme.wireColorScheme.onBackground,
-                text = item.title.asString(),
-                modifier = Modifier.padding(start = dimensions().spacing8x)
-            )
+            Row {
+                Text(
+                    style = MaterialTheme.wireTypography.body01,
+                    color = MaterialTheme.wireColorScheme.onBackground,
+                    text = title,
+                    modifier = Modifier.padding(start = dimensions().spacing8x)
+                )
+            }
         },
-        actions = Icons.Filled.ChevronRight.Icon(),
-        clickable = clickable
+        actions = {
+            trailingIcon?.let {
+                Icon(
+                    painter = painterResource(id = trailingIcon),
+                    contentDescription = "",
+                    tint = MaterialTheme.wireColorScheme.onSecondaryButtonEnabled,
+                    modifier = modifier
+                        .defaultMinSize(80.dp)
+                        .clickable(onIconPressed)
+                        .then(modifier)
```

Comment:
Now `modifier` is used twice 😄 Depending on the order it should be either:
```
modifier = modifier
    .defaultMinSize(80.dp)
    .clickable(onIconPressed)
```
or 
```
modifier = modifier
    .defaultMinSize(80.dp)
    .clickable(onIconPressed)
    .then(modifier)
```
In the second one, new `Modifier` is created and the existing `modifier` is used after all modifiers are set.